react/features/speaker-stats/functions.ts

```diff
@@ -208,3 +213,62 @@ export function resetHiddenStats(state: IReduxState, stats?: Object) {
 
     return resetStats;
 }
+
+/**
+ * Gets the current duration of the conference.
+ *
+ * @param {IState} state - The redux state.
+ * @returns {number | undefined} - The duration in milliseconds or undefined.
+ */
+export function getCurrentDuration(state: IReduxState) {
+    const startTimestamp = getConferenceTimestamp(state);
+
+    return startTimestamp ? Date.now() - startTimestamp : undefined;
```

Comment:
should be `null` instead of `undefined` as it's an intentional absence of the value